src/hashtable.py

```diff
@@ -80,32 +80,35 @@ def _hash_mod(self, key):
         return self._hash(key) % self.capacity
 
     def insert(self, key, value):
-        '''
-        Store the value with the given key.
-
-        # Part 1: Hash collisions should be handled with an error warning. (Think about and
-        # investigate the impact this will have on the tests)
 
-        # Part 2: Change this so that hash collisions are handled with Linked List Chaining.
-
-        Fill this in.
-        '''
-        # SOLUTION:
         index = self._hash_mod(key)
 
```

Comment:
Extremely readable code. Nice of use temp variable to actively distinguish insertions.